binderhub/builder.py

```diff
@@ -250,7 +250,8 @@ def escape(s):
             push_secret=push_secret,
             builder_image=self.settings['builder_image_spec'],
             memory_limit=self.settings['build_memory_limit'],
-            docker_host=self.settings['build_docker_host']
+            docker_host=self.settings['build_docker_host'],
+            git_credential=provider.git_credentials
```

Comment:
Is there a reason you call it `git_credential` (singular) above but `git_credentials` (plural) here?